Automattic/jetpack
# PHPCS Changed: The Next Generation

Third (or fifth?) time is the charm.

DO NOT MERGE UNTIL AFTER https://github.com/Automattic/jetpack/pull/13975 LANDS AND THE BETA BUILDER HAS BEEN UPDATED.

#### Changes proposed in this Pull Request:
* PHPCS processing on changed lines only.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* n/a

#### Testing instructions:
* On this branch, make changes to a whitelisted file and a non-whitelisted file ( anything in the packages folder for a whitelist or `uninstall.php` for non-whitelist. ( ` // This is a fine change.` or ` // This will throw an error`
* Attempt to commit (`yarn install` must have been run before at some point)
* Confirm that the precommit hook prevents it.

#### Proposed changelog entry for your changes:
* n/a
